### DISCUSSION TITLE
Format Implementations

### DISCUSSION THREAD

**Discussion by PhoneDroid** [temper-studios/graph-file-format#2] [Ideas]
<br>

I was wondering if you were planning on listing libraries written in
other languages that implement parsing / stringification of your format.

<br>

**Comment by temper-studios** [comment]
Yeah I can do that.

**Reply by PhoneDroid** [reply]
https://github.com/OmegaTools/GRAPH

**Comment by temper-studios** [comment]
If they do do that, I recommend they just wrap this code and call into it. That is one of the reason it is written in C. A lot of problems can be solved that way.

**Reply by PhoneDroid** [reply]
Well that wouldn't be very fun though ..

**Reply by temper-studios** [reply]
Where do people usually list these implementations? In the readme?

**Reply by PhoneDroid** [reply]
Some just place a list in the README, others use a separate file.

In the beginning, I would just place it in the README, however
to keep it nice and short I would move it as soon as the README
gets too stuffed and just link to it.

If you want to be fancy, you could also use the languages icons as
links and have dedicated pages, once there are more implementations.

( If you need help with that, let me know )
